website/content/en/preview/upgrade/upgrade-ref.md

```diff
@@ -0,0 +1,597 @@
+---
+title: "Karpenter Upgrade Reference"
+linkTitle: "Karpenter Upgrade Reference"
+weight: 10
+description: >
+  API information for upgrading Karpenter
+---
+
+Significant changes to the Karpenter APIs and various compatibility issues have been introduced in Karpenter v0.32.
+In this release, Karpenter APIs have advanced to v1beta1, in preparation for Karpenter v1 in the near future.
+The v1beta1 changes are meant to simplify and improve ease of use of those APIs, as well as solidify the APIs for the v1 release.
+Use this document as a reference to the changes that were introduced in the current release and as a guide to how you need to update the manifests and other Karpenter objects you created in previous Karpenter releases.
+
+The [Karpenter Upgrade Guide}(]({{< relref "upgrade-guide" >}}) steps you through the process of upgrading Karpenter for the latest release.
+For a more general understanding of Karpenter compatibility issues, see the ({{< relref "#compatibility" >}}) section of this document.
+
+# Karpenter Migration Information
+
+Use the information below to help migrate your Karpenter v1alpha assets to v1beta1.
+
+### Custom Resource Definition (CRD) Upgrades
+
+Karpenter ships with a few Custom Resource Definitions (CRDs). Starting with v0.32, CRDs representing Provisioners (karpenter.sh_provisioners.yaml), Machines (karpenter.sh_machines.yaml) and AWS Node Templates (karpenter.k8s.aws_awsnodetemplates.yaml) are replaced by those for Node Pools, Node Claims, and EC2 Node classes, respectively.
+You can find these CRDs on [Karpenter APIs CRD](https://github.com/aws/karpenter/tree/main/pkg/apis/crds) page.
+
+The [Karpenter Upgrade Guide}(]({{< relref "upgrade-guide" >}}) describes how to install the new CRDs using `kubectl`. However, you can instead install those CRDs as an independent helm chart [karpenter-crd](https://gallery.ecr.aws/karpenter/karpenter-crd), that can be used by Helm to manage the lifecycle of these CRDs.  See [source](https://github.com/aws/karpenter/blob/main/charts/karpenter-crd) for the source code,
```

Comment:
Should this be part of the upgrade guide? I recalled us originally talking about adding this to the FAQ

Reply:
I thought it went better here, since I don't think people would look for that information in the FAQ, but might run into it here as an alternative.